README.md

```diff
@@ -229,6 +229,37 @@ Specifies target architectures when using the multiplatform build CLI goal.
 
 *Default:* None
 
+#### wheelDependencies ####
+
+Optional set of wheel dependencies to retrieve from poetry cache. This allows previously cached external 
+wheel dependencies to be copied into a given target directory if it exists in poetry cache. This logic 
+depends on wheels to have first been cached by `cacheWheels` habushu-maven-plugin configuration and executes
+during the VALIDATE maven phase. Warnings will be logged if the specified wheel isn't found. 
+```xml
+  <plugin>
+      <groupId>org.technologybrewery.orphedomos</groupId>
+      <artifactId>orphedomos-maven-plugin</artifactId>
+      <executions>
+          <execution>
+              <id>retrieve-wheels</id>
+              <phase>validate</phase>
+              <goals>
+                  <goal>retrieve-wheels</goal>
+              </goals>
+              <configuration>
+                  <wheelDependencies>
+                      <wheelDependency>
+                          <artifactId>{artifact-id}</artifactId>
+                          <targetDirectory>${wheelOutputDirectory}</targetDirectory>
+                      </wheelDependency>
+                  </wheelDependencies>
+              </configuration>
+          </execution>
+      </executions>
+  </plugin>
+```
+#### managedDependencies ####
+
```

Comment:
Q: Is managedDependencies an empty header section? Does it make sense to add a "Default" line to wheelDependencies like all the other sections have?

Reply:
Good catch, this was incorrectly included. I also added a default line.

Reply:
We could probably do more to match the format of the others as well.

- Extract just the `<configuration>` block and put it under `*Example:*`
- Add `*POM Option:* <wheelDependencies>`
- Add `*CLI Option:* `